OutOfSchool/OutOfSchool.WebApi.Tests/Services/CompanyInformationServiceTests.cs

```diff
@@ -0,0 +1,193 @@
+
+        [SetUp]
+        public void SetUp()
+        {
+            var builder =
+                new DbContextOptionsBuilder<OutOfSchoolDbContext>().UseInMemoryDatabase(
+                    databaseName: "OutOfSchoolTestDB");
+
+            options = builder.Options;
+            context = new OutOfSchoolDbContext(options);
+
+            logger = new Mock<ILogger<CompanyInformationService>>();
+            mapper = new Mock<IMapper>();
+            repository = new EntityRepository<CompanyInformation>(context);
+            service = new CompanyInformationService(repository, logger.Object, mapper.Object);
+
+            SeedDatabase();
+        }
+
+        [Test]
+        [TestCase(CompanyInformationType.AboutPortal)]
+        [TestCase(CompanyInformationType.SupportInformation)]
+        public async Task GetByType_WhenTypeIsValid_ReturnsEntity(CompanyInformationType type)
+        {
+            // Arrange
+            Expression<Func<CompanyInformation, bool>> filter = p => p.Type == type;
+            var expected = await repository.GetByFilter(filter, "CompanyInformationItems").ConfigureAwait(false);
+
+            mapper.Setup(m => m.Map<CompanyInformationDto>(It.IsAny<CompanyInformation>())).Returns(expected.FirstOrDefault().ToModel());
+
+            // Act
+            var result = await service.GetByType(type).ConfigureAwait(false);
+
+            // Assert
+            Assert.AreEqual(expected.FirstOrDefault().Type, result.Type);
+        }
+
+        [Test]
+        [TestCase(CompanyInformationType.LawsAndRegulations)]
+        public async Task GetByType_WhenTypeIsValid_EntityNotExists(CompanyInformationType type)
+        {
+            // Arrange
+            Expression<Func<CompanyInformation, bool>> filter = p => p.Type == type;
+            var expected = await repository.GetByFilter(filter, "CompanyInformationItems").ConfigureAwait(false);
+
+            mapper.Setup(m => m.Map<CompanyInformationDto>(It.IsAny<CompanyInformation>())).Returns(expected.FirstOrDefault().ToModel());
+
+            // Act
+            var result = await service.GetByType(type).ConfigureAwait(false);
+
+            // Assert
+            Assert.IsFalse(expected.Any());
+            Assert.IsNull(result);
+        }
+
+        [Test]
+        [TestCase(CompanyInformationType.AboutPortal)]
+        public async Task Update_UpdatesExistedEntity(CompanyInformationType type)
+        {
+            // Arrange
+            var changedEntity = GetEntityDto(type);
+
+            // Act
+            mapper.Setup(m => m.Map<IEnumerable<CompanyInformationItem>>(It.IsAny<IEnumerable<CompanyInformationItemDto>>())).Returns(changedEntity.ToDomain().CompanyInformationItems);
+            mapper.Setup(m => m.Map<CompanyInformationDto>(It.IsAny<CompanyInformation>())).Returns(changedEntity);
+
+            var result = await service.Update(changedEntity, type);
+
+            // Assert
+            Assert.That(changedEntity.Title, Is.EqualTo(result.Title));
```

Comment:
This set of `Assert`s is duplicated in next test method. 2 times duplication is moderately ok, but for 3rd and subsequent time consider to reuse extracted helper method that perform these asserts in one place. 

Moreover it looks like you have another helper - to assert equality of `CompanyInformationItem`.

And also consider collection equality asserts provided by framework or write your own (or maybe any is already written by other devs - I don't remember if this is a case).